config/giantswarm-vault-access/manager_init_container_patch.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
```

Comment:
shouldn't the name be more descriptive? this looks too generic as it could by any controller

Reply:
Same applies: https://github.com/giantswarm/capi-migration/pull/23#discussion_r603289915